src/components/accordion.ts

```diff
@@ -0,0 +1,78 @@
+import {Component, View, NgClass, OnDestroy} from 'angular2/angular2';
+
+@Component({
+  selector: 'ngb-accordion, [ngb-accordion]',
+  properties: ['onlyOneOpen: closeOthers'],
+  host: {'class': 'panel-group'}
```

Comment:
This makes the compiler cry, both here and in the dropdown.

I think that the syntax is correct in here, by having both key and value inside quotes. The dropdown is not using quotes for `class` and I think it should. Anyhow the `d.ts` seems to be broken. that or the doc in it is deprecated.

I say to ignore this, wait for .38 and see if any of the two gets fixed and change the other.